src/aurora/actionscript/asbuffer.cpp

```diff
@@ -464,7 +464,20 @@ void ASBuffer::actionCallMethod(AVM &avm) {
 }
 
 void ASBuffer::actionEnumerate2() {
-	// TODO
+	if (!_stack.top().isObject())
+		throw Common::Exception("actionEnumerate2: not an object");
+
+	ObjectPtr object = _stack.top().asObject();
+	_stack.pop();
+
+	// push null as the end of the enumeration.
```

Comment:
Please start a comment with an upper-case letter. And no period at the end of simple single-line comments